src/sentry/lang/native/appconnect.py

```diff
@@ -257,6 +258,9 @@ class BuildInfo:
     # Apple naming calls this the "bundle_version".
     build_number: str
 
+    # The date and time the build was uploaded to App Store Connect.
+    uploaded_date: DateTimeField
```

Comment:
Hmm, this is not a database model.  I think you should use a plain `datetime.datetime` instance here.

Reply:
a `datetime` is also what you're putting there now even if i read it correctly.